wolox-books/src/app/screens/BooksList/components/shopping-list/shopping-list.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="modal-window">
+  <div class="shopping-list">
+    <div class="shopping-header">
+      <h1 class="shopping-title" i18n="Cart title|Shopping cart title@@cartTitle">Your shopping cart</h1>
+      <button type="button" class="close-icon" (click)="closeModal.emit()"></button>
+    </div>
+    <ul class="books-list" *ngIf="books.length > 0; else emptyCart">
+      <li *ngFor="let book of books; let i = index" class="book-item">
+        <span class="book-description">{{ book.title }} - {{ book.author }}</span>
+        <button type="button" class="delete-icon" (click)="removeBook(i)"></button>
```

Comment:
Use the book.id instead of index to remove the book.

Reply:
I already discussed this with @mcallegari10 in this comment: https://github.com/wolox-training/jr-angular/pull/25#discussion_r426883215 . In other words I would have to increment the logic of the reducer to use the ID, because the ID is different than the index in the array of the store, I don't know if it would better or clearer than just using the current index of each position in the array